raddb/clients.conf

```diff
@@ -136,6 +136,21 @@ client localhost {
 	#
 #	shortname = localhost
 
+	#
+	#  virtual_server:: Set the virtual server
+	#
+#	virtual_server = default
+
+	#
+	#  response_window:: How long the client has to respond.
+	#
+#	response_window = 0
+
+	#
+	#  track_connections:: Allow connected sockets to be set on a per-client basis.
```

Comment:
I don't understand what this means, and if I don't, people reading the docs probably won't either.  Could you seek clarification as to what this config item does from @alandekok